content/docs/1.6.0/advanced-resources/deploy/node-selector.md

```diff
@@ -43,20 +43,20 @@ You need to set node selector for both types of components. See more details bel
    >   systemManagedComponentsNodeSelector: "label-key1:label-value1"
    >  ```
 
-
 ### Setting up Node Selector After Longhorn has been installed
 
 > **Warning**:
 > * Since all Longhorn components will be restarted, the Longhorn system is unavailable temporarily.
-> * Make sure all Longhorn volumes are `detached`. If there are running Longhorn volumes in the system, this means the Longhorn system cannot restart its components and the request will be rejected.
+> * When all Longhorn volumes are detached, the customized setting is immediately applied to the system-managed components.
+> * When one or more Longhorn volumes are still attached, the customized setting is applied to the Instance Manager only when no engines and replica instances are running. You are required to reconfigure the setting after detaching the remaining volumes. Alternatively, you can wait for the next setting synchronization, which will occur in an hour.
 > * Don't operate the Longhorn system while node selector settings are updated and Longhorn components are being restarted.
 
 1. Prepare
    * If you are changing node selector in a way so that Longhorn cannot run on some nodes that Longhorn is currently running on,
      you will lose the volume replicas on those nodes.
      Therefore, It is recommended that you evict replicas and disable scheduling for those nodes before changing node selector.
      See [Evicting Replicas on Disabled Disks or Nodes](../../../nodes-and-volumes/nodes/disks-or-nodes-eviction) for more details about how to do this.
```

Comment:
This is incorrect anymore. Why wasn't it removed? Please review others as well.

>If you are changing node selector in a way so that Longhorn cannot run on some nodes that Longhorn is currently running on,
     you will lose the volume replicas on those nodes.
     Therefore, It is recommended that you evict replicas and disable scheduling for those nodes before changing node selector.
     See [Evicting Replicas on Disabled Disks or Nodes](../../../nodes-and-volumes/nodes/disks-or-nodes-eviction) for more details about how to do this.

Reply:
Removed and checked other danger zone settings. Thanks